scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -322,7 +322,7 @@ call PopulateFilterSetInfo {
       }
     }
 
-    call PopulateFilterSetInfo as PopulateFilterSetInfoCLassic {
+    call PopulateFilterSetInfo as PopulateFilterSetInfoClassic {
```

Comment:
does this change need to get propagated anywhere else?

Reply:
nah, it isn't needed.  Just changing it here is sufficient!